plans/example/manifest.toml

```diff
@@ -20,6 +20,9 @@ enabled = true
 [runners."local:docker"]
 enabled = true
 
+[runners."local:docker".additional_envs]
+TESTGROUND_HELLO_MESSAGE = "Hello, Docker."
+
 [runners."local:exec"]
 enabled = true
 
```

Comment:
(commenting here to get a thread)

> How do you actually pass run config values when running a plan via CLI? There is run-cfg but I do not see anything using it? Are there examples of this, as I suppose that would be the way to pass custom run cfgs, e.g. our custom environment variables.
> 
> Could you also explain me that @laurentsenta, as I would want to do that for my browser plan I imagine, in between different tests on same image?

At the moment, you would use a composition file, then pass the parameters there, for example:

https://github.com/testground/testground/blob/9c13f3bbe956c3cc29591db4b8b3c63b6fa1f6dc/plans/_integrations_mixed_builders/_compositions/issue-1357-mix-builder-configuration.toml#L22-L23

Reply:
Can you show me how that would work? As I am not sure how to go from a composition file parameter to a runner config value?!